include/zephyr/net/ocpp.h

```diff
@@ -141,12 +145,25 @@ struct ocpp_cp_info {
 	char *meter_type;	/**< Main power meter type */
 };
 
+#if defined(CONFIG_NET_SOCKETS_SOCKOPT_TLS)
```

Comment:
we would typically not put an include guard here - only the one for the struct field is necessary